src/pre_commit_terraform/terraform_docs_replace.py

```diff
@@ -53,12 +53,13 @@ def invoke_cli_app(parsed_cli_args: Namespace) -> ReturnCodeType:
         'https://github.com/antonbabenko/pre-commit-terraform/issues/248'
         '#issuecomment-1290829226',
         category=UserWarning,
+        stacklevel=1,
```

Comment:
Document the reasoning
```suggestion
        stacklevel=1,  # FIXME: figure out how to suppress in tests
```

Reply:
Documented. No need to figure out that.
[`3132f17` (#863)](https://github.com/antonbabenko/pre-commit-terraform/pull/863/commits/3132f1719de112b7f5354eea7001d399a6b5d37f)

Reply:
It might be useful to know for any future occurrences.